app/templates/dayview.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+        <link rel="stylesheet" href="{{ url_for('static', path='/dayview.css') }}">
+        <link href="https://fonts.googleapis.com/css2?family=Assistant&family=Open+Sans&family=Roboto&family=Roboto+Condensed&display=swap" rel="stylesheet">
+        <title>dayview</title>
+    </head>
+    <body>
+        <div id="toptab" class="d-flex justify-content-around sticky-top">
+            <a href="/closethedayview" title="close day view"><img class="pb-1" src="{{ url_for('static', path='/images/icons/close_sidebar.svg')}}" width="15em" height="15em"></a>
+            <span class="month fw-bold text-white">{{month}}</span>
+            <span class="day fw-bold text-white">{{day}}</span>
+        </div>
+        <div class="scedual">
+            <div class="container times bg-primeary position">
+                {% for i in range(24)%}
+                    <div class="row bg-transparent hourmark">
+                    {% set i = i|string() %}
+                    {{i.zfill(2)}}:00
+                    </div>
+                {% endfor %}
+            </div>
+            <div class="eventgrid">
+                {% for event, attr in events %}
+                    {% set totaltime = 'visible'%}
+                    {% if attr.length < 60 %}
+                            {% set size = '0.6em' %}
```

Comment:
Prefer to set `size` to be a class name. Try to separate the CSS from the HTML as much as possible :)